src/workloads/execution/ColumnstoreIndexes.yml

```diff
@@ -0,0 +1,467 @@
+      {"$limit": 5000}
+    ]
+  allowDiskUse: true
+  cursor: {batchSize: 2000}
+
+TopTenantsByNoOfEventsQuery: &TopTenantsByNoOfEventsQuery
+  aggregate: *Collection
+  pipeline:
+    [
+      {
+        "$match": {
+          "created_at": {"$gte": {"$date": "2020-03-09T01:18:32Z"}},
+          "resource": {"$nin": ["DashboardItem", "EmbeddedChart", "Error"]},
+          "metadata.target": {"$nin": [null]},
+          "metadata.tenant_id": {"$nin": [null]}
+        }
+      },
+      {
+        "$group": {
+          "_id": {"__alias_0": "$metadata.target", "__alias_1": "$metadata.tenant_id"},
+          "__alias_2": {"$sum": 1}
+        }
+      },
+      {
+        "$project": {
+          "_id": 0, "__alias_0": "$_id.__alias_0", "__alias_1": "$_id.__alias_1", "__alias_2": 1
+        }
+      },
+      {"$project": {"color": "$__alias_0", "x": "$__alias_2", "y": "$__alias_1", "_id": 0}},
+      {"$limit": 50000}
+    ]
+  allowDiskUse: true
+  cursor: {batchSize: 2000}
+
+
+# Defines test operations.
+TestOperations: &TestOperations
+- OperationMetricsName: DateFieldProjectionQuery
+  OperationName: RunCommand
+  OperationCommand: *DateFieldProjectionQuery
+- OperationMetricsName: DistinctStringFieldProjectionQuery
+  OperationName: RunCommand
+  OperationCommand: *DistinctStringFieldProjectionQuery
+- OperationMetricsName: DuplicatedStringFieldProjectionQuery
+  OperationName: RunCommand
+  OperationCommand: *DuplicatedStringFieldProjectionQuery
+- OperationMetricsName: CountPerActionQuery
+  OperationName: RunCommand
+  OperationCommand: *CountPerActionQuery
+- OperationMetricsName: TotalTenantCreatedQuery
+  OperationName: RunCommand
+  OperationCommand: *TotalTenantCreatedQuery
+- OperationMetricsName: ActiveCloudUsersLastMonthQuery
+  OperationName: RunCommand
+  OperationCommand: *ActiveCloudUsersLastMonthQuery
+- OperationMetricsName: VersionsHistogramQuery
+  OperationName: RunCommand
+  OperationCommand: *VersionsHistogramQuery
+- OperationMetricsName: UniqueUsersPerMonthQuery
+  OperationName: RunCommand
+  OperationCommand: *UniqueUsersPerMonthQuery
+- OperationMetricsName: TopErrorsQuery
+  OperationName: RunCommand
+  OperationCommand: *TopErrorsQuery
+- OperationMetricsName: NumberOfChartsPerTypeQuery
+  OperationName: RunCommand
+  OperationCommand: *NumberOfChartsPerTypeQuery
+- OperationMetricsName: EventsByVersionQuery
+  OperationName: RunCommand
+  OperationCommand: *EventsByVersionQuery
+- OperationMetricsName: BrowserUsageByDistinctUserQuery
+  OperationName: RunCommand
+  OperationCommand: *BrowserUsageByDistinctUserQuery
+- OperationMetricsName: TopTenantsByNoOfEventsQuery
+  OperationName: RunCommand
+  OperationCommand: *TopTenantsByNoOfEventsQuery
+
+# Defines how many times each test operation is executed.
+TestRepeatCount: &TestRepeatCount 10
+
+Actors:
+# The 'WarmupCache1' actor warms up the buffer cache.
+- Name: WarmupCache1
+  Type: RunCommand
+  Database: *db
+  Thread: 1
+  Phases:
+  - Repeat: 2
```

Comment:
Suggestion: [OnlyActiveInPhases](https://github.com/mongodb/genny/blob/master/src/workloads/scale/LoadTest.yml#L46-L58) might cut down on the verbosity a bit (and insulate you from any later changes).

Reply:
Good to know such thing exists!